mountpoint-s3-client/src/s3_crt_client.rs

```diff
@@ -543,18 +547,6 @@ impl ObjectClient for S3CrtClient {
     ) -> ObjectClientResult<PutObjectResult, PutObjectError, Self::ClientError> {
         self.put_object(bucket, key, params, contents).await
     }
-
-    async fn get_object_attributes(
-        &self,
-        bucket: &str,
-        key: &str,
-        max_parts: Option<usize>,
-        part_number_marker: Option<usize>,
-        object_attributes: &[ObjectAttribute],
-    ) -> ObjectClientResult<GetObjectAttributesResult, GetObjectAttributesError, Self::ClientError> {
-        self.get_object_attributes(bucket, key, max_parts, part_number_marker, object_attributes)
-            .await
-    }
```

Comment:
I believe this breaks the build, probably need to rebase your code to current main.